docs/kyma-environment-broker/03-03-hyperscaler-account-pool.md

```diff
@@ -0,0 +1,25 @@
+# Hyperscaler Account Pool
+​
+When provisioning clusters through Gardener with the Runtime Provisioner, the Kyma Environment Broker (KEB) requires a hyperscaler (GCP, Azure, AWS, etc.) account/subscription, but setting it up is outside of the Runtime Provisioner's scope. In order for the user to be able to easily provision Kyma clusters, Hyperscaler Account Pool was created. 
```

Comment:
I'd turn it around a bit. 

- To provision clusters through Gardener using the Runtime provisioner, the KEB requires a hyperscaler account. Managing the available hyperscaler accounts is not in the scope of the KEB. Instead, the available accounts are stored/handled by the HAP.